common/src/main/java/net/adoptopenjdk/icedteaweb/jvm/JvmUtils.java

```diff
@@ -54,71 +72,188 @@ private static boolean isValidStartingArgument(final String argument) {
-            "-XX:-ForceTimeHighResolution", /* use low resolution (default) */};
+                "-d32", /* use a 32-bit data model if available */
+                "-client", /* to select the client VM */
+                "-server", /* to select the server VM */
+                "-verbose", /* enable verbose output */
+                "-version", /* print product version and exit */
+                "-showversion", /* print product version and continue */
+                "-help", /* print this help message */
+                "-X", /* print help on non-standard options */
+                "-ea", /* enable assertions */
+                "-enableassertions", /* enable assertions */
+                "-da", /* disable assertions */
+                "-disableassertions", /* disable assertions */
+                "-esa", /* enable system assertions */
+                "-enablesystemassertions", /* enable system assertions */
+                "-dsa", /* disable system assertione */
+                "-disablesystemassertions", /* disable system assertione */
+                "-Xmixed", /* mixed mode execution (default) */
+                "-Xint", /* interpreted mode execution only */
+                "-Xnoclassgc", /* disable class garbage collection */
+                "-Xincgc", /* enable incremental garbage collection */
+                "-Xbatch", /* disable background compilation */
+                "-Xprof", /* output cpu profiling data */
+                "-Xdebug", /* enable remote debugging */
+                "-Xfuture", /* enable strictest checks, anticipating future default */
+                "-Xrs", /* reduce use of OS signals by Java/VM (see documentation) */
+                "-XX:+ForceTimeHighResolution", /* use high resolution timer */
+                "-XX:-ForceTimeHighResolution", /* use low resolution (default) */
+                "-XX:+PrintGCDetails", /* Gives some details about the GCs */
+                "-XX:+PrintGCTimeStamps", /* Prints GCs times happen to the start of the application */
+                "-XX:+PrintHeapAtGC", /* Prints detailed GC info including heap occupancy */
+                "-XX:+PrintTenuringDistribution", /* Gives the aging distribution of the allocated objects */
+                "-XX:+TraceClassUnloading", /* Display classes as they are unloaded */
+                "-XX:+CMSClassUnloadingEnabled", /* It needs to be combined with -XX:+CMSPermGenSweepingEnabled */
+                "-XX:+CMSIncrementalPacing", /* Automatic adjustment of the incremental mode duty cycle */
+                "-XX:+UseConcMarkSweepGC", /* Turns on concurrent garbage collection */
+                "-XX:-ParallelRefProcEnabled",
+                "-XX:+DisableExplicitGC", /* Disable calls to System.gc() */
+                "-XX:+UseG1GC",
+                "-XX:+HeapDumpOnOutOfMemoryError",
+                "-XX:-TransmitErrorReport",
+                "-XstartOnFirstThread",
```

Comment:
can you please move this up before the `-XX:` block

Reply:
The order is old props at top and new props at bottom. Order maintained from Java 8 doc.